pkgs/servers/monitoring/grafana/default.nix

```diff
@@ -31,6 +31,7 @@ buildGoPackage rec {
     tar -xvf $srcStatic
     mkdir -p $bin/share/grafana
     mv grafana-*/{public,conf,tools} $bin/share/grafana/
+  '' + lib.optionalString phantomJsSupport ''
```

Comment:
For what exactly do we need `phantomjs2`? Isn't the project dead anyway?

Reply:
Your right: https://grafana.com/docs/grafana/latest/guides/whats-new-in-v6-4/

Is everyone OK to juste remove it altogether ?

Should I remove the phantomjs2 package as well ?